R/testGeno.R

```diff
@@ -8,13 +8,13 @@
 
 # E an environmental variable for optional GxE interaction analysis.
 testGenoSingleVar <- function(nullmod, G, E = NULL, test = c("Score", "Score.SPA", "BinomiRare", "CMP"),
-                              recalc.pval.thresh = 1, GxE.return.cov = FALSE){
+                              recalc.pval.thresh = 1, approx.score.SE = FALSE, GxE.return.cov = FALSE){
     test <- match.arg(test)
     calc.score <- test %in% c("Score", "Score.SPA") | (recalc.pval.thresh < 1)
 
-    if (isNullModelSmall(nullmod) && (calc.score || !is.null(E))) {
-        stop("small null model cannot be used with options provided")
-    }
+    # if (isNullModelSmall(nullmod) && (calc.score || !is.null(E))) {
+    #     stop("small null model cannot be used with options provided")
+    # }
```

Comment:
I moved this check into assocTestSingle and integrated it with some other checks. If that's a problem, we can move them back. Also, this check had to be updated because you do (can) use a small null model when using approx.score.SE

Reply:
having it in assocTestSingle is better; we don't need to run the check for every iteration